src/main/java/org/wso2/carbon/connector/ldap/SearchEntry.java

```diff
@@ -50,6 +51,17 @@ public void connect(MessageContext messageContext) throws ConnectException {
         String filter = (String) getParameter(messageContext, LDAPConstants.FILTERS);
         String dn = (String) getParameter(messageContext, LDAPConstants.DN);
         String returnAttributes[] = ((String) getParameter(messageContext, LDAPConstants.ATTRIBUTES)).split(",");
+
+        int limit = 0;
+        String searchLimit = (String) getParameter(messageContext, "limit");
```

Comment:
Move "limit" property to the constant file and make use of that

Reply:
Done